src/applications/static-pages/download-1095b/tests/fixtures/500.json

```diff
@@ -1,5 +1,5 @@
 {
   "status": 500,
-  "error": "Internal Server Error",
+  "errors": "Internal Server Error",
```

Comment:
This doesn't impact anything but it is more accurate because the server returns errors, not error.